app/controllers/api/account_images_controller.rb

```diff
@@ -1,19 +1,16 @@
 # encoding: utf-8
 
-class Api::AccountImagesController < Api::BaseController
-  include Announce::Publisher
-
+class Api::AccountImagesController < Api::BaseImagesController
   api_versions :v1
+  represent_with Api::ImageRepresenter
 
   filter_resources_by :account_id
 
-  announce_actions decorator: Api::Msg::ImageRepresenter, subject: :image
-
-  represent_with Api::ImageRepresenter
-
   child_resource parent: 'account', child: 'image'
 
   def after_original_destroyed(original)
```

Comment:
If I am honest, I don't really know what any of this is doing. I assume it is so that you can write to the attributes as though they are a nested part of the main object.

Reply:
Well, since we don't use this right now, will leave as an exercise for the first app implementing account/user images to figure out.